iothub/device/src/IotHubConnectionStringBuilder.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.Azure.Devices.Client
     /// <summary>
     /// Builds a connection string for the IoT Hub service based on the properties populated by the user.
     /// </summary>
-    public sealed class IotHubConnectionStringBuilder
+    public sealed class IotHubConnectionStringBuilder : IDisposable
```

Comment:
This is now IDisposable and could be breaking change but something necessary. I want to call this out so that we can make a decision otherwise if necessary.

Reply:
It is an additive change.